templates/workloads/partials/cf/publish.yml

```diff
@@ -0,0 +1,31 @@
+{{- if .Publish}}
+{{- range $topic := .Publish.Topics}}
+{{.Name}}SNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: !Join ['', [!Ref AppName, '-', !Ref EnvName, '-', {{.Name}}]]
+      
+  {{.Name}}SNSTopicPolicy:
+    Type: AWS::SNS::TopicPolicy
+    DependsOn: {{.Name}}SNSTopic
+    Properties:
+      Topics: 
+        - !Ref ${AWS::StackName}-{{.Name}}-SNSTopic
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          {{- range $worker := .AllowedWorkers}}
+          - Effect: Allow
+            Principal:
+              Service: sqs.amazonaws.com
+            Action: 
+              - sns:Subscribe
+            Resource: !Sub 'arn:aws:sqs:${AWS::Region}:${AWS::AccountId}:*'
```

Comment:
```suggestion
            Resource: !Sub 'arn:${AWS::Partition}:sqs:${AWS::Region}:${AWS::AccountId}:*'
```

Reply:
Neat, I didn't know this was a thing! Adjusted.